src/main/java/com/github/invictum/dto/AbstractDto.java

```diff
@@ -20,12 +20,28 @@
 
     private String extractData(Object object, Field attribute) {
         attribute.setAccessible(true);
+        Object value;
         try {
-            return (String) attribute.get(object);
-        } catch (IllegalAccessException e) {
+            value = attribute.get(object);
+        } catch (IllegalAccessException var4) {
             LOG.error("Failed to get data for {} attribute", attribute);
+            return null;
         }
-        return null;
+
```

Comment:
1) fixed;
2) depending of multiply return there no need in if ... else clause, because in case of condition of if is true - we will exit the method;
3) Not implemented default return, because of problem with object.toString().